csharp/src/Google.Protobuf/Google.Protobuf.csproj

```diff
@@ -1,13 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>C# runtime library for Protocol Buffers - Google's data interchange format.</Description>
     <Copyright>Copyright 2015, Google Inc.</Copyright>
     <AssemblyTitle>Google Protocol Buffers</AssemblyTitle>
     <VersionPrefix>3.11.0-rc0</VersionPrefix>
-    <LangVersion>6</LangVersion>
+    <LangVersion>6.0</LangVersion>
```

Comment:
I will try to check if using C# 7.2 for the entire Google.Protobuf project would be acceptable (this could make things simpler). This is only for the runtime Google.Protobuf,  the generated code needs to be able to build with an old version of C# (unless the protoc flag to enable span support is used).